rsconf/package_data/vm_devbox/start.sh.jinja

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -eou pipefail
+
+vm_devbox_first_start() {
+    if [[ -e Vagrantfile ]]; then
+        return
+    fi
+    curl '{{ rsconf_db.installers_url }}' | vagrant_dev_vm_devbox=1 bash -s vagrant-sirepo-dev '{{ this.vm_hostname}}'
+}
+
+vm_devbox_set_forwarded_port() {
+    python <<'EOF'
+import re
+import subprocess
+
+_FILE = "Vagrantfile"
+_FORWARDED_PORT_PREAMBLE = 'config.vm.network "forwarded_port"'
+_FORWARDED_PORT_FULL = (
+    "{}, guest: {{ this.ssh_port }}, host: {{ this.ssh_port }}".format(
+        _FORWARDED_PORT_PREAMBLE
+    )
+)
+r = False
+with open(_FILE, "r+") as f:
+    c = f.read()
+    if _FORWARDED_PORT_PREAMBLE not in c:
+        c = re.sub(
+            r"^\s*(config\.vm\.hostname.*)$",
+            r"\1\n{}".format(_FORWARDED_PORT_FULL),
+            c,
+            flags=re.MULTILINE,
+        )
+        r = True
+    elif _FORWARDED_PORT_FULL not in c:
+        c = re.sub(
+            r"^\s*{}.*$".format(_FORWARDED_PORT_PREAMBLE),
+            _FORWARDED_PORT_FULL,
+            c,
+            flags=re.MULTILINE,
+        )
+        r = True
+    f.seek(0)
+    f.write(c)
+    f.truncate()
+if r:
+    subprocess.check_call(("vagrant", "reload"))
```

Comment:
seems like this could be a systemctl restart. Just needs ExecReload or maybe ExecStop. We can turn off auto restarts except when singled and this would make it better integrated in the rsc.